ServerCore/Pages/Teams/TeamCompositionSummary.cshtml

```diff
@@ -7,102 +7,92 @@
     ViewData["PlayRoute"] = "/Teams/Index";
 }
 
-<!DOCTYPE html>
+<style>
+    td, th {
+        text-align:center;
```

Comment:
I think we have css elsewhere

Reply:
I didn't end up adding or removing anything from this file functionally, so this CSS was just what we had before.